src/Lib/LanguagesLib.php

```diff
@@ -677,8 +678,9 @@ public static function getLanguageDirection($lang) {
             "phn",
             "jpa",
             "bal",
-            'hbo',
-            "ajp"
+            "hbo",
+            "ajp",
+            "ayl',
```

Comment:
Thanks for trying to make the quotes consistent. :+1: But you missed the closing quote of `ayl`: it’s a single quote instead of a double one.

Reply:
Thanks!